lib/winappdriver.js

```diff
@@ -130,8 +132,38 @@ class WinAppDriver extends events.EventEmitter {
   }
 
   async startSession (caps) {
+
+    const createSessionTimeout = caps.createSessionTimeout || this.createSessionTimeout;
+    log.debug(`Start WinAppDriver session during createSessionTimeout = '${createSessionTimeout}' ms.`);
```

Comment:
Then I would rather rephrase the log line to `Starting WinAppDriver session. Will timeout in ... ms`

Reply:
Done